py/fiberassign/targets.py

```diff
@@ -107,7 +107,7 @@ def default_sv1_sciencemask():
     sciencemask |= sv1_mask["QSO"].mask
     sciencemask |= sv1_mask["BGS_ANY"].mask
     sciencemask |= sv1_mask["MWS_ANY"].mask
-    sciencemask |= sv1_mask["SECONDARY_ANY"].mask
+    sciencemask |= sv1_mask["SCND_ANY"].mask
```

Comment:
Apologies for the non-backwards compatible change.  To support new-fiberassign on old-targetfiles, let's do
```python
if "SCND_ANY" in sv1_mask.names():
    sciencemask |= sv1_mask["SCND_ANY"].mask
else:
    sciencemask |= sv1_mask["SECONDARY_ANY"].mask
```
and ditto for `desi_mask`

Reply:
Ok, no problem.